src/lib/containers/widgets/query-filter/RangeFacetFilter.tsx

```diff
@@ -80,51 +80,52 @@ export const RangeFacetFilter: React.FunctionComponent<RangeFacetFilterProps> =
         onClick={(isCollapsed: boolean) => setIsCollapsed(isCollapsed)}
         facetAliases={facetAliases}
       ></FacetFilterHeader>
+      <div style={{ display: isCollapsed ? 'none' : 'block' }}>
```

Comment:
Did you make any changes in this file or just updating the spacing?

Reply:
this was the only line that I actually changed, everything else was spacing.